index.html

```diff
@@ -8,17 +8,32 @@
     <title>Awesome Books</title>
   </head>
   <body>
-    <header>
+    <header class="header" id="header">
       <div class="header-top">
         <h1 class="title1">Awesome Books</h1>
       </div>
+      <ul class="nav-bar">
+        <li class="nav-bar-li" id="booklist">Book List</li>
+        <li class="nav-bar-li" id="addbook">Add book</li>
+        <li class="nav-bar-li" id="contactinfo">Contact info</li>
+      </ul>
     </header>
-    <section>
-      <h2>Selected Books</h2>
+    <div class="time"></div>
+    <section class="dis-none contactinfo-section">
+      <h2>Contact Info</h2>
+      <p>
+        <span> Authors</span>: Ersoy,Reda,Didi,<br />
+        <span> Organisation</span>: microverse,<br />
+        <span> Website</span> :microverse.org, <br />
+        All rights reserved
+      </p>
+    </section>
+    <section class="dis-none" id="booklistsection">
+      <h2 class="title2">Your book list</h2>
       <div class="books-wrapper" id="books-wrapper"></div>
```

Comment:
- [ ] The styling of your _books list rows_ background color does not conform to the database table alternate rows color, as can be seen in the original wireframe below. Kindly ensure to improve upon this aspect.

Your design:
![image](https://user-images.githubusercontent.com/12745474/143275615-0e3970ae-0d56-474b-82bd-d43a4a9907d2.png)

**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/134323111-24a1a54d-374a-4197-bdc5-7ddbbcc89636.png)